core/list.go

```diff
@@ -255,6 +255,8 @@ func (lb *ListBase) Init() {
 		// absorb horizontal here, vertical in view
 		s.Overflow.X = styles.OverflowAuto
 		s.Grow.Set(1, 1)
+		svi := lb.This.(Lister)
```

Comment:
`svi` is not an appropriate name for a `Lister`, even if you copied it from some other place; always correct these old names when you come across them. `ls` is standard.

Reply:
fixed